sunpy/instr/goes.py

```diff
@@ -122,15 +137,15 @@ def temp_em(goeslc, abundances="coronal", download=False, download_dir=DATA_PATH
 
     Returns
     -------
-    goeslc.data.temperature : pandas.core.series.Series
-        Array of temperature values [MK]
-
-    goeslc.data.em : pandas.core.series.Series
-        Array of volume emission measure values [10**49 cm**-3]
+    lc_new : `sunpy.lightcurve.LightCurve`
+        Contains same metadata and data as input GOESLightCurve with the
+        following two additional data columns:
```

Comment:
I didn't put line breaks in because I thought Travis or Sphinx might complain.  But line breaks would be good is that is acceptable.  I didn't know you could italicise.  How do you do that?